app/src/main/res/layout/main_fragment.xml

```diff
@@ -11,7 +11,6 @@
         android:id="@+id/message"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="MainFragment"
```

Comment:
Unnecessary distraction on the UI.

Reply:
Yes... a user will not know what that is.